randovania/games/prime1/logic_database/Phendrana Drifts.txt

```diff
@@ -2511,7 +2511,7 @@ Extra - aabb: [329.07877, -1248.0734, -17.693329, 398.72052, -1143.9733, 42.7440
               Space Jump Boots
               Any of the following:
                   # https://www.youtube.com/watch?v=1-jXmt9cIzI
-                  Slope Jump (Advanced)
+                  R-Jump (Advanced) and Slope Jump (Advanced)
```

Comment:
I'd say `R-Jump (Intermediate) OR L-Jump (Advanced)` since an R-Jump isn't necessary and an L-Jump can be used instead. The hard part of this trick comes from the slope jump, not the R/L-Jump.